app/concepts/matestack/ui/core/q/q.rb

```diff
@@ -0,0 +1,5 @@
+module Matestack::Ui::Core::Q
+  class Q < Matestack::Ui::Core::Component::Static
+
```

Comment:
Here, you probably need to merge the `cite` option into `@tag_attributes`

Reply:
OK, got you.